src/find_github_repo.ipynb

```diff
@@ -509,30 +510,98 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {
     "collapsed": false
    },
    "outputs": [],
-   "source": []
+   "source": [
+    "from github import Github\n",
+    "\n",
+    "g = Github(\"MelanieIStefan\", \"hn8PJuM\")\n"
```

Comment:
Is this a private API key to access your account? Probably don't want to include it in the public record here right?.